s3/README.md

```diff
@@ -0,0 +1,225 @@
+# Spice.ai Quickstart Tutorials using S3
+
+## For public S3 bucket
+
+**Step 1.** Initialize a Spice application, then start the Spice runtime.
+
+```bash
+spice init s3-demo-project
+```
+
+```bash
+cd s3-demo-project
+spice run
+```
+
+**Step 2.** Configure the dataset to connect to S3:
+
+```bash
+spice dataset configure
+```
+
+Enter the name of the dataset:
+
+```bash
+dataset name: (s3-demo-project)  taxi_trips
+```
+
+Enter the description of the dataset:
+
+```
+description: taxi trips in s3
+```
+
+Specify the location of the dataset:
+
+```bash
+from: s3://spiceai-demo-datasets/taxi_trips/2024/
+```
+
+**LIMITATION:** Spice.ai currently does not support discovering nested directories in S3. Please ensure that the parquet files are in the root of the specified S3 path.
```

Comment:
I don't think we need limitations in quickstarts, I would just add this to the S3 section of docs.